common/config.go

```diff
@@ -78,9 +67,6 @@ func LoadFromCommandLine(config *Config) (*Config, error) {
 
 	flagSet.StringVar(&config.ConfigFile, "conf-file", "", "File path of configuration file")
 
-	flagSet.StringVar(&config.PluginDir, "plugin-dir", config.PluginDir, "Docker plugin directory where driver socket is created")
-	flagSet.StringVar(&config.DriverName, "driver-name", config.DriverName, "Name of Infoblox IPAM driver")
```

Comment:
Each instance of the driver define a specific set of IPAM configurations. So, if the deployment requires different set of networks with different IPAM configurations, then the user would need to run different instances of the driver, each with a specific ipam-driver name.